kaggle/api/kaggle_api_extended.py

```diff
@@ -290,8 +290,17 @@ class KaggleApi(KaggleApi):
     MAX_NUM_INBOX_FILES_TO_UPLOAD = 1000
     MAX_UPLOAD_RESUME_ATTEMPTS = 10
 
-    config_dir = os.environ.get('KAGGLE_CONFIG_DIR') or os.path.join(
-        expanduser('~'), '.kaggle')
+    config_dir = os.environ.get('KAGGLE_CONFIG_DIR')

```

Comment:
Should this be removed, so that this PR just marks the release?

Reply:
Yes, you're right.  I need to fix another problem, so I'll probably have to do another release anyway.